dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/common/logging/LogUtils.java

```diff
@@ -264,36 +264,64 @@ protected static Logger createLogger(final Class<?> cls,
                 try {
                     b = BundleUtils.getBundle(cls, bundleName);
                 } catch (MissingResourceException rex) {
-                    //ignore
                 }
             }
             if (b != null) {
                 b.getLocale();
             }
 
             if (loggerClass != null) {
-                try {
-                    Constructor<?> cns = loggerClass.getConstructor(String.class, String.class);
-                    if (name == null) {
+                // Default logger of Liberty is WsLogger. 
+                // CXFLogger is inherited from this WsLogger class to preserve same behavior
+                // except the one that we aimed to change. Which is to add class name to log records
+                if (loggerClass.equals(WsLogger.class)) {
+                    try {
```

Comment:
Suggest changing to be consistent
```suggestion
                   // Liberty Change Start: Adding class name as a parameter
                    try {
```

and then put an end block at the bottom of the try/catch - rather than having them on each line.